pkgs/development/python-modules/mulpyplexer/default.nix

```
@@ -0,0 +1,21 @@
+{ buildPythonPackage
+, fetchPypi
+, pkgs
```

please avoid importing pkgs :)
```suggestion
, lib
```